### ISSUE TITLE
getRankInGroup and getRankNameInGroup produce obscure errors

### ISSUE THREAD

**Issue by Neztore** [suufi/noblox.js#181]
If you run them and get an error returned, you get:
```
Unhandled rejection TypeError: bodyObject.filter is not a function
```
Why?
We expect Roblox to return an array which we run .filter on.
In the event of an error, Roblox returns an object with an errors key. Object.filter is not a function, hence the issue.

Could be fixed by adding an error check.

**Comment by j8cksxn** [comment]
This is happening to me.

**Comment by Neztore** [comment]
Then you're providing an invalid value. Probably not a user id.

**Comment by Neztore** [comment]
Fixed in #194
